tests/test.yml

```diff
@@ -56,6 +56,7 @@
     superset_white_label_use_filepaths: False
     superset_additional_pip_packages: ["pandas==0.23.4", "sqlalchemy==1.2.18", "ipdb"]
     superset_install_additional_pip_packages: True
+    superset_flask_app_mutator: True
```

Comment:
Could we please set this as `false` by default?

Reply:
It is set to false in the main.yml file. Or is there somewhere else ?

Reply:
I think it should be false both here and in main.yml - because for it to work it depends on a `superset_patchup` version that has the views module

Reply:
cool, I'll just remove it from the test then